src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -46,22 +40,33 @@ private String readInputFile(String pathToFile) {
     }
 
     private String calculateStatistic(String inputString) {
-        String[] arrayData = inputString.split(System.lineSeparator());
+        String[] statistic = inputString.split(System.lineSeparator());
         int suppliesCount = 0;
         int buyCount = 0;
-        for (String simpleLine : arrayData) {
+        for (String simpleLine : statistic) {
             String[] lineToArray = simpleLine.split(WORDS_DIVIDER);
-            if (lineToArray[FIRST_ELEMENT_INDEX].equals(SUPPLY_OPERATION)) {
-                suppliesCount += Integer.parseInt(lineToArray[SECOND_ELEMENT_INDEX]);
+            if (lineToArray[OPERATION_TYPE_INDEX].equals(SUPPLY_OPERATION)) {
+                suppliesCount += Integer.parseInt(lineToArray[AMOUNT_INDEX]);
             } else {
```

Comment:
what if file structure will change? here you should look specifically for "buy"
```suggestion
            } else if (lineToArray[OPERATION_TYPE_INDEX].equals(BUY_OPERATION) {
```

Reply:
done